src/corelibs/U2Core/src/util/U2FeatureUtils.cpp

```diff
@@ -94,7 +94,12 @@ QList<U2Feature> U2FeatureUtils::exportAnnotationDataToFeatures(const QList<Shar
 
         // add sub-features
         if (annotation->location->isMultiRegion()) {
-            U2FeatureUtils::addSubFeatures(annotation->location->regions, annotation->location->strand, feature.id, rootFeatureId, dbiRef, os);
+            U2FeatureUtils::createSubFeatures(annotation->location->regions.mid(1),
```

Comment:
Do not get it - why did you remove the first element?

Reply:
The first region is always saved as a part of the parent/top-level annotation.